src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java

```diff
@@ -126,11 +131,7 @@ public synchronized CircuitBreaker reset() {
   private synchronized void sendUpdateOnEventBus() {
     String address = options.getNotificationAddress();
     if (address != null) {
-      vertx.eventBus().publish(address, new JsonObject()
-          .put("name", name)
-          .put("state", state)
-          .put("failures", failures)
-          .put("node", vertx.isClustered() ? ((VertxInternal) vertx).getClusterManager().getNodeID() : "local"));
+      vertx.eventBus().publish(address, metrics.toJson());
```

Comment:
Shouldn't we use `send` instead? I'm trying to figure out how this would work if we have a clustered Vert.x behind a load balancer.